tickets/src/components/content/CreateContent.js

```diff
@@ -163,7 +163,7 @@ export class CreateContent extends Component {
                     <SaveButton type="submit">Save Event</SaveButton>
                     <Text>
                       Your event link: <br />
-                      <Cta>
+                      <Cta href={lockAddress}>
```

Comment:
Should this open in a new tab?

Reply:
It probably should — I think an overall better approach will be to make `EventUrl` return an `<a>` tag with that baked in.